New package: DifferentiationInterface v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: DifferentiationInterface
- Repository: https://github.com/gdalle/DifferentiationInterface.jl
- Created by: @gdalle
- Version: v0.1.0
- Commit: e3bc10d5190b6e1c21788b1a55e403490edfd482
- Git reference: HEAD
- Description: An interface to various automatic differentiation backends in Julia.

gdalle commented:
Blocking the registration while we talk things out with the AbstractDifferentiation.jl crew

JuliaHubRegistrator committed:
New package: DifferentiationInterface v0.1.0

UUID: a0c0ee7d-e4b9-4e03-894e-1c5f64a51d63
Repo: https://github.com/gdalle/DifferentiationInterface.jl.git
Tree: 84dc07b6f73d07cb2157c50cc691c1ed46b629e7

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09
